### PR TITLE
NodeJs v10 support

### PR DESCRIPTION
**Pull request by mhoeger**
Expands supported versions to be 8.x and 10.x

Note: test failures still need to be addressed when using v10.0.0 from this issue: https://github.com/nodejs/node/issues/20258

**Must consume latest proto file before pulling into functions-host: https://github.com/Azure/azure-functions-language-worker-protobuf/commit/a42a580e5211f89d5cc605a869a48425fccbd880**

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -4,6 +4,7 @@
         "target": "es6",
         "noImplicitAny": false,
         "strict": true,
+        "strictNullChecks": false,
```
Comment (pragnagopa) [inline tsconfig.json]: should we keep the default as it might be safer?

**Comment by paulbatum** [comment]
Just curious - why does the upgrade to node 10 result in a breaking change in the proto file?

**Comment by mhoeger** [comment]
We need to upgrade protobufjs to be able to use node 10, and upgrading protobufjs resulted in a few breaking changes. One of them is that we used to call the package "FunctionRpc" and the service "FunctionRpc" in our proto file. This now doesn't compile from type errors due to the naming conflict

**Comment by paulbatum** [comment]
got it, thanks!

**Pull request merged**